tensorflow/python/kernel_tests/control_flow_ops_py_test.py

```diff
@@ -570,6 +570,94 @@ def testCond_7(self):
       r = control_flow_ops.cond(pred, fn1, fn2)
       self.assertAllEqual([11, 12], sess.run(r))
 
+  # Test for list
```

Comment:
nit: you can remove these comments above the tests, since the test name gives you the same info now